### ISSUE TITLE
Scale panel

### ISSUE THREAD

**Issue by josvanr** [NsCDE/NsCDE#8]
Hello!

Have been trying NsCDE, nice work, thnx ! :)

I was wondering if there is a way to scale up the size of the panel, because it is a bit small on the high dpi screens of today. There is a script 'run_scaled' that allows individual applications to do this via xpra. Don't know if this is possible for the panel. Is this an fvwm module?

Jos v Riswick

**Comment by NsCDE** [comment]
Hello Jos!

Yes, Front Panel is FvwmButtons FVWM module. If you click upper leftmost button on FP you can temporary set new size with "Size" of the FP which can be put back with "Reposition Window" or Ctrl+Esc key binding. Small size adaptations can be tolerated visually, but swallowed applets and icons will not scale, and there will be big gaps between FP icons which are by default 48x48 px. Not really nice.

I have been using xpra, but never tried what you have described here or scaled windows in that way.

I have been thinking about making 64x64 px icon version of the Front Panel with bigger width and height, but this should also support finding 64x64 icons for "Copy to Main Panel" right mouse click on the subpanel item and probably subpanels should then be 48x48 instead of 32x32.

Also, 64x64 icon size is not common in the original CDE icon set (foo.l.pm == 48x48), so this will require graphical work at least for the default icons (terminal, file manager, style manager ...), it cannot be simply resized with ImageMagick because this will look ugly. Also there are FP clock, calendar applets ...

In short, days and days of work, plus some user variable which will select bigger FP and Subpanels. In plan, but not now.

For me, on 2560x1440 it is still really ok, on 1920x1080 optimal, but I can imagine that on higher resolutions, CDE Front Panel experience may become unauthentic. :-)

Another approach can be to add two launchers left and right (== 4) to get wider FP, but height will still be a problem.
